CHANGES.rst

```diff
@@ -112,6 +112,11 @@ astropy.io.registry
 astropy.io.votable
 ^^^^^^^^^^^^^^^^^^
 
+- Changed ``pedantic`` argument to ``verify`` and change it to have three
```

Comment:
I think this should go into the AP changes section

Reply:
I thought it was already?